x/ccv/provider/keeper/proposal.go

```diff
@@ -320,10 +319,13 @@ func (k Keeper) GetPendingConsumerAdditionProp(ctx sdk.Context, spawnTime time.T
 	chainID string) (prop types.ConsumerAdditionProposal, found bool) {
 	store := ctx.KVStore(k.storeKey)
 	bz := store.Get(types.PendingCAPKey(spawnTime, chainID))
-	if len(bz) == 0 {
```

Comment:
Should we panic for `len(bz) == 0`?

Reply:
GetPendingConsumerAdditionProp is only used in testing. So it's not really relevant.